src/util/misc.rs

```diff
@@ -317,8 +317,8 @@ mod tests {
 
         let secp = secp256k1::Secp256k1::new();
         let message = "rust-bitcoin MessageSignature test";
-        let msg_hash = super::signed_msg_hash(&message);
-        let msg = secp256k1::Message::from(msg_hash);
+        let msg_hash = super::signed_msg_hash(message);
+        let msg = secp256k1::Message::from_slice(&msg_hash).expect("message");
```

Comment:
What? This looks like a step back.

Reply:
Wow, I don't know how I missed this.

Reply:
Oh boy, that's a bad mistake, thanks @Kixunil. I did not check over my rebase well enough.